src/client/Client.js

```diff
@@ -402,10 +408,26 @@ class Client extends BaseClient {
 
     const query = new URLSearchParams({
       client_id: this.application.id,
-      scope: 'bot',
     });
 
-    if (options.permissions) {
+    const scopes = options.scopes;
+    if (typeof scopes === 'undefined') {
+      throw new TypeError('INVITE_MISSING_SCOPES');
+    }
+    if (!Array.isArray(scopes)) {
+      throw new TypeError('INVALID_TYPE', 'scopes', 'Array of Invite Scopes', true);
+    }
+    const requiredScope = scopes.find(scope => scope === 'bot' || scope === 'applications.commands');
+    if (!requiredScope) {
```

Comment:
```suggestion
    if (scopes.every(scope => scope !== 'bot' && scope !== 'applications.commands')) {
```

Reply:
I'd suggest moving those hardcoded scopes into an array or something, so we can extend them more easily in the future.

Reply:
This comment seems counter to your suggestion though...